querybook/server/lib/ai_assistant/base_ai_assistant.py

```diff
@@ -167,6 +174,30 @@ def _get_query_execution_error(self, query_execution: QueryExecution) -> str:
 
         return error[:1000]
 
+    def handle_ai_command(self, command_type: str, payload: dict = {}):
+        data_cell_id = payload.get("data_cell_id")
```

Comment:
what if data_cell_id is not provided? also what if it is not needed? i think the payload should be parsed per command